utils/m0reportbug

```diff
@@ -105,14 +105,15 @@ main() {
                 warn "$IN_DIR: no such directory"
                 continue
             }
-            [[ -L $IN_DIR ]] && {
+            # This check is commented because R2 does not have symlinks for /var/motr unlike R1.
+            #[[ -L $IN_DIR ]] && {
                 # As per our design, after failover softlink /var/motr is
                 # created to points /var/motr{1-2}, and since our INPUT_DIR
                 # is globe pattern (/var/motr*) we end up collecting m0traces
                 # and cores twice unnecessary, hence skipped data collection
                 # if IN_DIR is a soft link.
-                continue
-            }
+                #continue
+            #}
```

Comment:
Looks like the entire section is commented out. Remove it if it is not required.

Reply:
This check is not required in R2 design, but it is required for R1. So I thought of keeping it commented for future reference.

Reply:
@bdekvadiya, please remove this commented out portion.

Reply:
[suggestion]
can we keep  this change specific to dev env only ?
 if it is dev environment then only follow the symlinks otherwise leave it.

[Suggestion]  
If we are not going to merge this into R1 code base ever again then better to remove this completely instead of commenting it.